functions/lib/sendgrid/commons.py

```diff
@@ -0,0 +1,20 @@
+from lib.constants import db
+from lib.logging import Logger
+
+CREATE_EVENT_EMAIL_TEMPLATE_ID = "d-63eb299010344bcf951af2537c42a410"
```

Comment:
is it possible to put this somewhere hidden, i think its bad to have this out

Reply:
email templates should be public domain knowledge, i don't think these need to be kept private.. unless sendgrid advises somwhere?